examples.md

```diff
@@ -119,3 +120,23 @@ uses: actions/cache@preview
     restore-keys: |
       ${{ runner.os }}-mix-
 ```
+
+## Python - pip
+
+```yaml
+- uses: actions/cache@preview
+  if: startsWith(runner.os, 'Linux')
+  with:
+    path: ~/.cache/pip
+    key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+    restore-keys: |
+      ${{ runner.os }}-pip-
+
+- uses: actions/cache@preview
+  if: startsWith(runner.os, 'macOS')
+  with:
+    path: ~/Library/Caches/pip
+    key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+    restore-keys: |
+      ${{ runner.os }}-pip-
+```
```

Comment:
There should also probably be an example for windows here. The path I believe is `path: ~\AppData\Local\pip\Cache` from [pip docs](https://pip.pypa.io/en/stable/reference/pip_install/#caching).

Reply:
Confirmed: https://github.com/hugovk/pypistats/runs/288196149

Reply:
```yml
- uses: actions/cache@preview
  if: startsWith(runner.os, 'Windows')
  with:
    path: ~\AppData\Local\pip\Cache
    key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
    restore-keys: |
      ${{ runner.os }}-pip-
```